lms/static/js/views/fields.js

```diff
@@ -129,7 +154,8 @@
                 this.lastSuccessMessageContext = context;
 
                 setTimeout(function () {
-                    if ((context === view.lastSuccessMessageContext) && (view.getNotificationMessage() === successMessage)) {
+                    if ((context === view.lastSuccessMessageContext) &&
+                        (view.getNotificationMessage() === successMessage)) {
```

Comment:
Looks like we use `successMessage` elsewhere, so I'd recommend using it in line 145 above instead of repeating the `getMessage()` call.